components/styles/styler.service.ts

```diff
@@ -62,4 +63,20 @@ export class HsStylerService {
       }),
     ];
   }
+
+  /**
+   * @function save
+   * @memberof hs.styler.controller
+   * @description Get current style variables value and style current layer accordingly
```

Comment:
Incorrect description. Would change tjhat ust dont want to interfeere with unexpected commits

Reply:
I have some commits on the go. I gonna squeeze this change between them 😉 Thanks for the tip.

Reply:
Corrected in cbdaab6